packages/rulesets/src/asyncapi/index.ts

```diff
@@ -282,6 +295,17 @@ export default {
         function: asyncApi2DocumentSchema,
       },
     },
+    'asyncapi-server-variables-defined': {
```

Comment:
I don't know if it's a good name, or maybe simpler `asyncapi-server-variables` will be better.

Reply:
Ditto prior comment - obv a bit subjective, but for the same reasons I think `asyncapi-server-variables` is a better choice.

Reply:
Thanks! Changed :)